### PR TITLE
[BIT] 转换能力补齐：bincount、multiply、Tensor.[rank、log_normal_、tile]

### PR DESCRIPTION
**Pull request by yuwu46**
## [BIT] 转换能力补齐：bincount、multiply、Tensor.[rank、log_normal_、tile]

本次共补充了 `paddle.bincount`、`paddle.multiply`、`paddle.Tensor.rank`、`paddle.Tensor.log_normal_` 和 `paddle.Tensor.tile` 五个 API 的 paddle2torch 转换能力。具体情况如下：

| Paddle API 名称 | PyTorch API 名称 | 功能差异描述 | 配置撰写情况 | 备注 | API 对照表链接 |
|:------|:-----|:-------------|:-------------|:-----|:---------------|
| paddle.bincount | torch.bincount | 参数名不一致| 撰写 mapping 配置，若Paddle 中 `minlength` 参数为`Tensor[]`，则在torch中转为int| - | [对照表](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.bincount.html) |
| paddle.Tensor.rank | 无对应 PyTorch API | - | 编写 rule：`RankRule`，借助 `torch.Tensor.dim` 进行比较 | 需要额外规则 | - |
| paddle.Tensor.log_normal_ | torch.Tensor.log_normal_ | torch 参数更多 | 属于 rand_apis，撰写 mapping 配置，测试后 6 条配置均能正常转换（accuracy 报错），移入 tester/api_config/api_config_paddleonly_5.txt | 属于 rand_apis | [对照表](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.log_normal_.html) |
| paddle.Tensor.tile | torch.Tensor.tile | torch输入参数形式用法更多 | 撰写 mapping 配置,paddle做转化无需额外处理 | - | [对照表](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.tile.html) |
| paddle.multiply | torch.mul | torch 参数更多（out=None） | 撰写 mapping 配置 | - | [对照表](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.mul.html) |

除 `paddle.Tensor.log_normal_` 外(共6条配置),其余四个 API 原merged文件中**共52230条配置**，其中有三条重复配置
`paddle.Tensor.tile(Tensor([128, 1],"float32"), list[Tensor([],"int64"),], )`
`paddle.Tensor.tile(Tensor([1, 280, 376, 25, 3],"float32"), tuple(Tensor([],"int64"),1,1,1,1,), )`
`paddle.Tensor.tile(Tensor([256, 1],"float32"), list[Tensor([],"int64"),], )`

**剔除后共52227条配置**，其中有一条`paddle.Tensor.tile(Tensor([29151, 1, 32],"float16"), list[1,100,1,], )`为accuracy error,经反复测试并调整config_analyze.py中的初始化大小，依旧存在比较小的精度差异，其余"float16"类型的Tensor、list中存在大于100的数字则均可通过。相关信息已放入fresh_report和accuracy_error_5.txt中。
image
**其余52226条配置**均通过测试。
f74dc117e4b739c512cbf7286b009090

### CODE REVIEW

File: tester/paddle_to_torch/mapping.json [hunk 1]
```diff
@@ -85,6 +85,20 @@
...
+            "weights": "None",
+            "minlength": 0
+        },
+        "paddle_torch_args_map": {
+            "x": "input",
+            "weights": "weights"
+        },
+        "torch_kwargs": {
+            "minlength": "int({minlength}.item()) if torch.is_tensor({minlength}) else {minlength}"
+        }
```
Comment (cangtianhuang) [inline tester/paddle_to_torch/mapping.json:100]: int({minlength}.item()) 是否冗余，能否直接写成 {minlength}.item() ？

**Review by cangtianhuang — CHANGES_REQUESTED** [review]

File: tester/paddle_to_torch/rules.py [hunk 2]
```diff
@@ -2003,6 +2001,17 @@ def apply(self, paddle_api: str) -> ConvertResult:
         return ConvertResult.success(paddle_api, code, "result")
 
 # r
+class RankRule(BaseRule):
+    def apply(self, paddle_api: str) -> ConvertResult:
+        pre = f"tensor_for_rank = next(iter(kwargs.values()))"
+        core = f"result = torch.tensor(tensor_for_rank.ndim, dtype=torch.int32)"
```
[the remark this answers is not in the document]
Comment (yuwu46) [reply tester/paddle_to_torch/rules.py]: 已修改，感谢指正

**Review by cangtianhuang — APPROVED** [review]
LGTM
